app/main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
There is no need to define `skills` as a class attribute. Extend `self,skills` with required list in the `__init__` directly here and in the other classes as well